components/gcp/dataproc/create_cluster/component.yaml

```diff
@@ -0,0 +1,76 @@
+# Copyright 2018 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+name: dataproc_create_cluster
+description: |
+  Creates a DataProc cluster under a project.
+inputs:
+  - name: project_id
+    description: >-
```

Comment:
Just asking: Why do you put the description text on the new line? There is nothing wrong with that, but I'm trying to understand why.

Reply:
I want to break the text in multiple lines for better readability.